assets/runtime/config/gitlabhq/rack_attack.rb

```diff
@@ -10,13 +10,14 @@
   "#{Rails.application.config.relative_url_root}/api/#{API::API.version}/session",
   "#{Rails.application.config.relative_url_root}/users",
   "#{Rails.application.config.relative_url_root}/users/confirmation",
-  "#{Rails.application.config.relative_url_root}/unsubscribes/"
-
+  "#{Rails.application.config.relative_url_root}/unsubscribes/",
+  "#{Rails.application.config.relative_url_root}/import/github/personal_access_token"
 ]
 
 # Create one big regular expression that matches strings starting with any of
 # the paths_to_be_protected.
 paths_regex = Regexp.union(paths_to_be_protected.map { |path| /\A#{Regexp.escape(path)}/ })
+rack_attack_enabled = Gitlab.config.rack_attack.git_basic_auth['enabled']
 
 unless Rails.env.test?
```

Comment:
Guys, there's already a tagged v8.12.1, you might want to check it out: https://gitlab.com/gitlab-org/gitlab-ce/commit/afd43301c016553619b60d8db47537d935aad328